src/purifier/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Is there a reason to embed this project inside the remainder of amphtml? Or could it be a standalone repo?

Reply:
I think we should probably move towards that in the future, but it seems too tightly coupled with this repo right now (it imports various utils and the caja sanitizer [depends on things](https://github.com/ampproject/amphtml/blob/1afef3af8fadd8fc3e2fdc289287d65c34649807/src/sanitizer.js#L17-L26) that probably shouldn't be directly exported by the package). Maybe good candidate for a future fixit?